New package: AllenNeuropixels v0.1.0

JuliaRegistrator opened the pull request:
- Registering package: AllenNeuropixels
- Repository: https://github.com/brendanjohnharris/AllenNeuropixels.jl
- Created by: @brendanjohnharris
- Version: v0.1.0
- Commit: accfc31b3b5addd2bcddddc67f79ea073f86dee7
- Reviewed by: @brendanjohnharris
- Reference: https://github.com/brendanjohnharris/AllenNeuropixels.jl/issues/48#issuecomment-2301146046
- Description: Load, format, analyze, and plot the Allen Neuropixels datasets
<!-- bf0c69308befbd3ccf2cc956ac8a4671113b158fd87a6d4e741b0173dab051a48f7dc43e593ffff4357bc647a3a8359859c32ede8af5f449a6aa78f6bd8767af001fa282e771be12aff39f69d13b984c5df0bb6ce9e4b3b6e8f8238054be07ecaeb7e6999d409bbe908e23be1095e36d2c13c20dc502624ad9a9ea9645d69f0f92256b08cf765c88b4fea04b4b9ca5ce13abee222f2b4c123470cb627ac1873d7054397353ddaf506467b031466c493a078d183f2d9f0adb7e2f33847e45d246 -->

goerz commented:
Thank you for submitting your package! However, please make sure to add some documentation before registering. [At the very least](https://discourse.julialang.org/t/when-does-it-make-sense-to-post-a-new-package-to-the-julia-repository/108701/4), that would be a small usage example in the README. But based on the description, a Documenter-based API documentation seems more appropriate.

brendanjohnharris commented:
No worries, I've added a documenter page with some basic usage examples and lists of documented functions

JuliaRegistrator committed:
New package: AllenNeuropixels v0.1.0

UUID: c7dbd8d7-5c13-4dcb-90e2-bff79a096f56
Repo: https://github.com/brendanjohnharris/AllenNeuropixels.jl.git
Tree: eb80a230c9b6799eef7141248ffe05d02ed6d43e

Registrator tree SHA: 17aec322677d9b81cdd6b9b9236b09a3f1374c6a

goerz commented:
[noblock] Thanks! Make sure to add `[noblock]` to your comment

brendanjohnharris commented:
Cool, thanks! [noblock]
